docs/source/actions.rst

```diff
@@ -71,6 +71,52 @@ To execute an action manually, you can use ``st2 run <action with parameters>``
    # If you want to add a trace tag to execution when you run it, you can use:
    st2 run core.local cmd=date --trace-tag="simple-date-check-`date +%s`"
 
+Modification in Action Delete API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|st2| offers functionality to delete actions/workflows by invoking API. Prior to 3.6 release
+this API was only de-registering actions from database. In the 3.6 release, with kept backward
```

Comment:
Perhaps replace "In the 3.6 release, with kept backward compatibility this API was modified to delete related action files from disk as well."
to 
"In the 3.6 release, with backward compatibility retained this API was modified to optionally delete related action files from disk as well."